src/NuGet.Clients/NuGet.SolutionRestoreManager/Resources.resx

```diff
@@ -180,4 +180,4 @@ To prevent NuGet from restoring packages during build, open the Visual Studio Op
   <data name="SolutionIsNotSaved" xml:space="preserve">
     <value>Solution is not saved. Please save your solution before managing NuGet packages.</value>
   </data>
-</root>
\ No newline at end of file
+</root>
```

Comment:
It looks like only an empty line was added to this file. Please revert if that's the case.